modules/parquet/src/parquetjs/compression.ts

```diff
@@ -86,17 +86,17 @@ export function decompress(method: ParquetCompression, value: Buffer, size: numb
   return toBuffer(compressedArrayBuffer);
 }
 
-/**
+/*
  * Inflate a value using compression method `method`
- *
-export function inflate(method: ParquetCompression, value: Buffer, size: number): Buffer {
+ */
+export function inflate(method: ParquetCompression, value: Buffer): Buffer {
   if (!(method in PARQUET_COMPRESSION_METHODS)) {
     throw new Error(`invalid compression method: ${method}`);
   }
-
-  return PARQUET_COMPRESSION_METHODS[method].inflate(value, size);
+  // @ts-ignore
+  return PARQUET_COMPRESSION_METHODS[method].inflate(value);
 }
-
+/*
```

Comment:
Rebase? I think there are already commented out. They can just be cut.

Reply:
WIll rebase in next commit